test/acceptance/keywords/run_on_failure_only_once.txt

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Suite Setup     Run Keywords  Prefer Custom Keywords  Open Browser To Front Page
+Suite Teardown  Run Keywords  Close Browser  Set Debug Loglevel  Restore Old Search Order
+Test Teardown   Register Keyword To Run On Failure  Nothing
+Resource        ../resource.txt
+Library         CustomSeleniumKeywords
+
+*** Test Cases ***
+Run On Failure Keyword Only Called Once
```

Comment:
Actually, @zephraph, the reason this is in its own suite is because the keyword I am registering to run on failure is a custom keyword defined in a separate library. The Suite Setup commands used in `test/acceptance/keywords/run_on_failure.txt` will not work if I am importing that library, because it gets imported in addition to Selenium2Library, requiring the use of `Set Library Search Order` to be run before the Suite Setup commands, to disambiguate.